.github/workflows/build_and_test.yml

```diff
@@ -443,5 +444,7 @@ jobs:
       working-directory: ./ToyVpn-PcapPlusPlus
       run: |
         sed -i.bak "s|abiFilters.*$|abiFilters '${{ matrix.target }}'|g" app/build.gradle
+        sed -i.bak "s|gradle-6.5-bin|gradle-7.3-bin|g" gradle/wrapper/gradle-wrapper.properties
+        sed -i.bak "s|jcenter()|mavenCentral()|g" build.gradle
```

Comment:
I already merged https://github.com/seladb/ToyVpn-PcapPlusPlus/pull/4 so I guess this change is no longer needed?

Reply:
Correct let me remove it

Reply:
done